File: apps/server/src/main/java/com/zl/mjga/ApplicationService.java
package com.zl.mjga;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.modulith.Modulithic;
import org.springframework.scheduling.annotation.EnableScheduling;

@Modulithic
@SpringBootApplication
@ConfigurationPropertiesScan
@EnableScheduling
public class ApplicationService {

  public static void main(String[] args) {
    SpringApplication.run(ApplicationService.class, args);
  }
}

File: apps/server/src/main/kotlin/com/zl/mjga/ApplicationService.kt
package com.zl.mjga

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.modulith.Modulithic
import org.springframework.scheduling.annotation.EnableScheduling

@Modulithic
@SpringBootApplication
@ConfigurationPropertiesScan
@EnableScheduling
class ApplicationService

fun main(args: Array<String>) {
    runApplication<ApplicationService>(*args)
}
